packaging/installer/kickstart.sh

```diff
@@ -78,12 +123,14 @@ USAGE: kickstart.sh [options]
   --disable-cloud            Disable support for Netdata Cloud (default: detect)
   --require-cloud            Only install if Netdata Cloud can be enabled. Overrides --disable-cloud.
   --install <path>           Specify an installation prefix for local builds (default: autodetect based on system type).
+  --old-install <path>       Specify an old local builds installation prefix for uninstall/reinstall (if it's not default).
```

Comment:
Could we possibly change this to `--old-install-prefix`? I’m looking at probably going to the effort of renaming the exiting `--install` option (so that we can use that name eventually for other things and the meaning is not ambiguous) to `--install-prefix`, and I’d rather not have to rename multiple options at the same time.